inject-test/src/test/java/org/example/coffee/priority/base/CBasei.java

```diff
@@ -0,0 +1,15 @@
+package org.example.coffee.priority.base;
+
+import io.avaje.inject.Priority;
+
+import javax.inject.Singleton;
+
+@Singleton
+@Priority(1)
+public class CBasei implements BaseIface {
+
+  @Override
+  public String other() {
+    return "a";
```

Comment:
Minor nit, but should this be `return "c";`?

Reply:
Yes it should, it will be nice to fix it - thanks